presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOracleConfig.java

```diff
@@ -45,13 +46,15 @@ public void testExplicitPropertyMappings()
                 .put("oracle.timestamp.precision", "3")
                 .put("oracle.number.default-scale", "2")
                 .put("oracle.number.rounding-mode", "CEILING")
+                .put("oracle.fetchSize", "5000")
```

Comment:
We usually name configs with dashes instead of camel case. Can you make this `oracle.fetch-size`?

Reply:
done

Reply:
Thanks, please also update the test

Reply:
yep.. sorry..
the tipical, its only a literal, nothing else change :d
done